nbcelltests/shared.py

```diff
@@ -152,9 +156,29 @@ def is_empty(source):
     return len(parsed.body) == 0
 
 
-def cell_injected_into_test(test_lines):
-    for test_line in test_lines:
-        if test_line.strip().startswith(r"%cell"):
+CELL_INJ_TOKEN = r"%cell"
+
+
+def source2lines(source):
+    return source.splitlines(keepends=True)
+
+
+def lines2source(lines):
+    return "".join(lines)
+
```

Comment:
Good that these are now clear and in one place, and that they do the same thing as nbformat. But they should just be calls to nbformat. Either fix or add TODO.



https://github.com/jupyter/nbformat/blob/16032b927d19c2860efdd46762c3020761cb0154/nbformat/v4/rwbase.py#L20

Reply:
I don't think we can call out to nbfromat, as that will not be able to handle fields in metadata. Happy to leave as-is.